### ISSUE TITLE
Connecting to named MATLAB session

### ISSUE THREAD

**Issue by jonniedie** [JuliaInterop/MATLAB.jl#173]
It would be really nice to be able to connect to an open MATLAB session the way you can with Python. Or, in the very least, start a full MATLAB GUI session when starting MATLAB up from Julia.

**Comment by musm** [comment]
Yeah, problem is we are using the C api and it's severely limited compared to the C++ api
